core/app/models/spree/order_shipping.rb

```diff
@@ -70,7 +70,10 @@ def ship(inventory_units:, stock_location:, address:, shipping_method:,
       # TODO: make OrderShipping#ship_shipment call Shipment#ship! rather than
```

Comment:
This is my new favourite comment in Solidus. Props to @kennyadsl for managing to read it out without stumbling in the core team meeting today. 👏🏻 

<img width="1061" alt="CleanShot 2022-04-13 at 08 18 26" src="https://user-images.githubusercontent.com/471667/163214276-3e3653fe-1035-4b47-9e14-fe19b68ce0cd.png">

Reply:
You made my day